assets/src/models/detour.ts

```diff
@@ -1,64 +1,30 @@
 import { LatLngLiteral } from "leaflet"
 import { ShapePoint, Stop } from "../schedule"
 import { CreateDetourMachineInput } from "./createDetourMachine"
-import { array, Infer, nullable, number, string, type } from "superstruct"
+import { any, Infer, number, string, type } from "superstruct"
 
-export const SimpleDetour = type({
-  route: string(),
-  direction: string(),
-  name: string(),
-  intersection: string(),
+export const DetourWithState = type({
+  author: string(),
+  state: any(),
```

Comment:
Oh, I'm sorry I didn't see this sooner, but we don't need to define our internal structs with superstruct (and probably shouldn't)

We only need superstruct definitions for parsing boundries, so `DetourWithStateFromApi` should be `DetourWithStateData` and we should have a `type` or `interface` named `DetourWithState`

Reply:
I'm a little conflicted, on one hand, this PR is not introducing new structs, but just renaming them, on the other hand I'd prefer this not to slip longer just because "that's not what this PR is for".

If you want to do that in another PR I'd be fine with that and could call this non-blocking.

Reply:
Ahhhhhh I get it! Yeah I was very happy to make the fix in this PR. Thanks for catching